instrumentation/aws-sdk/aws-sdk-2.2/library-autoconfigure/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/autoconfigure/AwsSdkSingletons.java

```diff
@@ -25,18 +24,19 @@ private static class AwsSdkTelemetryFactory extends AbstractAwsSdkTelemetryFacto
     @Override
     protected List<String> getCapturedHeaders() {
       return ConfigPropertiesUtil.getList(
-          "otel.instrumentation.messaging.experimental.capture-headers", emptyList());
+          GlobalOpenTelemetry.get(), "messaging", "capture_headers/development");
     }
 
     @Override
     protected boolean messagingReceiveInstrumentationEnabled() {
       return ConfigPropertiesUtil.getBoolean(
-          "otel.instrumentation.messaging.experimental.receive-telemetry.enabled", false);
+              GlobalOpenTelemetry.get(), "messaging", "receive_telemetry/development", "enabled")
+          .orElse(false);
     }
 
     @Override
-    protected boolean getBoolean(String name, boolean defaultValue) {
-      return ConfigPropertiesUtil.getBoolean(name, defaultValue);
+    protected boolean getBoolean(boolean defaultValue, String... name) {
```

Comment:
can we make this the more readable

```suggestion
    protected Optional<Boolean> getBoolean(String... name) {
```

Reply:
turns out that the default value is always `false`, so we can remove the param

Reply:
removed the default value param